android/src/main/kotlin/com/airship/flutter/FlutterAutopilot.kt

```diff
@@ -64,7 +64,10 @@ class FlutterAutopilot : BaseAutopilot() {
         airship.pushManager.notificationListener = object : NotificationListener {
```

Comment:
You can only have a single notification listener. Instead of setting another one you should use the events from the proxy:

```
  MainScope().launch {
            // Background events will create a headless JS task in ReactAutopilot since
            // initialized wont be called until we have a JS task.
            EventEmitter.shared().pendingEventListener
                    .filter { it.type == WHATEVER }
                    .collect {
                        handleBackgroundMessage...
                    }
        }
```